npm/index.js

```diff
@@ -62,17 +10,13 @@ class CircomRunner {
         args,
         env,
         preopens = defaultPreopens,
-        bindings = defaultBindings,
         quiet = false,
     } = {}) {
-        if (!bindings.fs) {
-            throw new Error('You must specify an `fs`-compatible API as part of bindings')
-        }
         this.wasi = new WASI({
+            version: 'preview1',
             args: ['circom2', ...args],
             env,
             preopens,
-            bindings,
             quiet,
```

Comment:
Quiet flag should set the `stdout` and `stderr` descriptor to 1337. Right now the flag is misused